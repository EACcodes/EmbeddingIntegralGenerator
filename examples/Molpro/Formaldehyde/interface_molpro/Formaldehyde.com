***,C
print,basis,orbitals

symmetry,nosym
geometry={
4
angstrom
C   3.594222       6.218747       8.509041
O   2.916401       6.422257       7.534980
H   3.528797       5.268722       9.100681
H   4.337350       6.964926       8.891556
}

basis=cc-pVTZ

hf
en_no_emb = energy

{matrop
LOAD,one_int,H0
READ,emb_int,FILE=emb_ints.molpro
ADD,H0,one_int,emb_int
SAVE,H0,1200.1,h0
}

hf
en_emb = energy

show en_no_emb
show en_emb
