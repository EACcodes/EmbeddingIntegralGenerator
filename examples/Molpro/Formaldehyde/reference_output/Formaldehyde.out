
********************************************************************************
********************************************************************************
**********                                                            **********
                        EMBEDDING INTEGRAL GENERATOR                            
**********                                                            **********
********************************************************************************
********************************************************************************

********************************************************************************
**********                                                            **********
                        CONFIGURATION OF THIS CALCULATION                       

Keyword                                 Value
-------                                 -----
BASIS FILE                              basis_set_TZ
CARTESIAN                               0
EMBEDDING INTS REAL                     0
EMBEDDING POTENTIAL FILE                embpot.dat
GRID DIMENSIONS                         100  80  120  
LATTICE VECTORS                         14.3065  0  0  
                                        -7.15324  12.3898  0  
                                        0  0  17.0087  
NUM THREADS                             16
OUTPUT FORMAT                           MOLPRO
SHIFT VECTOR                            Not defined, use default:
                                        0  0  0  
XYZ FILE                                Formaldehyde.xyz
**********                                                            **********
********************************************************************************

Running with OpenMP using 16 threads
***********************
*                      
* Embedding Integrals  
*                      
***********************

  Read grid data and embedding potential wall time(s): 0.391 cpu time(s): 0.38

* Reciprocal space integration scheme
  Calculate embedding integrals, real space part wall time(s): 28.976 cpu time(s): 211.27
  Calculate embedding integrals, reciprocal space part wall time(s): 34.467 cpu time(s): 256.26

  Calculate embedding integrals, sorting and printing wall time(s): 34.497 cpu time(s): 256.69
TOTAL EXECUTION TIME  wall time(s): 34.903 cpu time(s): 257.07
